sdcm/commit_log_check_thread.py

```diff
@@ -176,13 +178,26 @@ def run_thread(self):
                 message=f"CommitLogCheckThread failed: {exc.__repr__()} with traceback {trace}").publish()
         self.log.debug("CommitLogCheckThread finished")
 
+    @staticmethod
+    def get_commit_log_directory_max_exceed_time(prometheus_response):
+        # get max time diff from response per shard(line in response)
+        exceed_time_list = []
+        for line in prometheus_response:
+            time_list = [item[0] for item in line["values"]]
```

Comment:
isn't that what you are trying to achieve ?
```python
max_time = max(item[0] -  line["values"][0][0] for item in line["values"])
```

Reply:
line["values"][0][0] - it is firs item 

I'm looking for the diff between min d max value 
ideally, it should be just ```line["values"][0][-1]-line["values"][0][0]`` but I have not found info that prometheus guarantees time ordering in response, so in finding max and min value by sorting

Reply:
I'm still not sure what's the calculation done in this function, and why the assumption the time would ever be bigger  than 60sec


those time are exactly the times prometheus is querying them, and if you aren't looking at the value of the metric, it don't mean anything about what was scylla doing or not doing, you should be looking at the value as well.

Reply:
i'm filtering values in request https://github.com/scylladb/scylla-cluster-tests/blob/master/sdcm/commit_log_check_thread.py#L64

so Prometheus' response has only bad values.

initially, it was an assumption that if we got any bad value from Prometheus' response - produce Error event
now, according to a discussion with @elcallio , it can be bad values for a short period. 
so I added such a gate for 1 minute per shard, per request for 10 minimum range with 1 sec discreteness

Reply:
according to @elcallio comment. it also needs to be limited to shart 0
```b.) Only on shard 0 in this case.```